tasks/task2/draw2_styled.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3

```

Comment:
This file looks like it contains a lot of the same functionality as the previous file. If you find you are re writing and re using the same code in multiple areas, I would recommend splitting it out and making it more modular.